### DISCUSSION TITLE
Sample SQLNet application with verification

### DISCUSSION THREAD

**Discussion by muratbedir** [zauberzeug/nicegui#3163] [Ideas / Feature Requests]
If there Need a full working sample sqlnet application that makes verification, multiple filed verifications filed size restrictions sample it will be very useful for beginners.

**Comment by rodja** [comment]
Could you please use a separate feature request for each topic. Eg. one for the pdf book and one for the sqlnet sample application? That will make the topics simpler to track.

**Reply by muratbedir** [reply]
Ok. thanks.
